guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -68,12 +70,26 @@ Then register the hosts with a secure `curl` command, such as:
 # curl -sS https://{foreman-example-com}/register ...
 ----
 +
-The following is an example of the `curl` command with the `--insecure` option:
+If you chose to use `wget`, register the hosts with a secure `wget` command, such as:
```

Comment:
Please swap the order; show the default command "curl" first, then "wget" second.

Reply:
The diff is a little bit confusing here. The order is

1.  curl secure
2. wget secure
3. gurl insecure
4. wget insecure

Reply:
I guess first curl https and http; then wget https http. not 100% which order is best.

Reply:
Agreed, I've switched the order